README.md

```diff
@@ -4,7 +4,27 @@
 
 Appcircle CLI is a unified tool for accessing the Appcircle platform features from the command line.
 
-# Installation
+## Table of contents
+
+- [Table of contents](#table-of-contents)
+- [Installation](#installation)
+- [Usage](#usage)
+   - [Commands](#commands)
+- [Environment Variables](#environment-variables)
+- [Interactive Mode](#interactive-mode)
+- [Logging requests](#logging-requests)
+- [Guides](#guides)
+  - [How to Configure your Appcircle CLI environment?](#how-to-configure-your-appcircle-cli-environment)
+  - [How to Trust your Self-Hosted Appcircle Certificate?](#how-to-trust-your-self-hosted-appcircle-certificate)
+  - [How to Connect your Appcircle Account within CLI?](#how-to-connect-your-appcircle-account-within-cli)
+  - [How to start a new build via the Appcircle CLI?](#how-to-start-a-new-build-via-the-appcircle-cli)
+  - [How to distribute an app via the Appcircle CLI?](#how-to-distribute-an-app-via-the-appcircle-cli)
+- [Contribution](#contribution)
+- [Publishing](#publishing)
+- [Jenkins pipeline](#jenkins-pipeline)
+- [Docker image](#docker-image)
```

Comment:
IMO, this list should have an active voice to better user-friendly

for example,

- Table of Contents
- Installation Instructions
- Usage Guidelines
   - Command List
- Environment Variable Settings
- Interactive Mode Details
- Request Logging
- Guides and Tutorials
   - Configuring Your Appcircle CLI 
   - Environment
   - Trusting Your Self-Hosted Appcircle
   - Certificate
   - Connecting Your Appcircle Account via CLI
   - Starting a New Build via the Appcircle CLI
   - Distributing an App via the Appcircle CLI
- How to Contribute
- Publishing Guidelines
- Jenkins Pipeline Integration
- Docker Image Usage